universal-application-tool-0.0.1/app/views/questiontypes/RepeaterQuestionRenderer.java

```diff
@@ -36,43 +37,70 @@ public RepeaterQuestionRenderer(ApplicantQuestion question) {
 
   @Override
   public Tag render(Messages messages) {
-    ContainerTag repeaterFields = div().withId(REPEATER_FIELDS_ID);
-    // TODO: add each answer as a repeaterField
-    repeaterFields.with(repeaterField(PLACEHOLDER, Optional.empty(), 0));
+    RepeaterQuestion enumeratorQuestion = question.createEnumeratorQuestion();
+    ImmutableList<String> entityNames = enumeratorQuestion.getEntityNames();
+
+    ContainerTag enumeratorFields = div().withId(ENUMERATOR_FIELDS_ID);
+    for (int index = 0; index < entityNames.size(); index++) {
+      enumeratorFields.with(existingEnumeratorField(Optional.of(entityNames.get(index)), index));
+    }
 
     return div()
         .withClasses(Styles.MX_AUTO, Styles.W_MAX)
         .with(
             div()
                 .withClasses(ReferenceClasses.APPLICANT_QUESTION_TEXT)
                 .withText(question.getQuestionText()),
+            div()
+                .withId(PLACEHOLDER_ID)
+                .withClass(Styles.HIDDEN)
```

Comment:
question: What is this div for and why is it hidden?

Reply:
Fixed!!! This is much more straightforward now.